tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.md

```diff
@@ -101,7 +101,7 @@ vic-machine create --name=<VCH_NAME> --target=<TEST_URL> \
 
 ### Expected Outcome
 * Deployment succeed
-* VCH is deployed with debug enabled
+* VCH is deployed with debug value set to -1
```

Comment:
with my above comment, we'd better not to show the debug value to -1 as it's not a valid value in usage